rmf_site_editor/src/site/group.rs

```diff
@@ -34,6 +34,10 @@ impl Members {
     pub fn iter(&self) -> impl Iterator<Item = &Entity> {
         self.0.iter()
     }
+
+    pub fn len(&self) -> usize {
```

Comment:
If you add `DerefMut` to the derives, I don't think we'll need these member functions.